wildnlp/datasets/conll.py

```diff
@@ -54,14 +54,17 @@ def load(self, path):
                 processed = self._process_sample(sample)
                 self._data.append(processed)
 
-    def apply(self, aspect, apply_to_ne=False):
+    def apply(self, aspect, apply_to_ne=False, apply_to_both=False):
```

Comment:
Maybe it would be better to rewrite it so for example there is the `apply_to` parameter that can take one of three values:
`NON_NE`, `NE`, `BOTH`? I think that when it's controlled by two variables it's a little bit confusing.